### DISCUSSION TITLE
Name the Timer

### DISCUSSION THREAD

**Discussion by nitsanavni** [felladrin/LinkedTimer#19] [Ideas]
### Is your feature request related to a problem? Please describe.

-

### Describe the solution you'd like

Allow the user to give the timer a name

**Comment by felladrin** [comment]
Thanks for bringing up this idea! I'd love to work on it when I have some free time!
If you thought about how it should work, let me know.
I'm thinking in adding a pencil icon near the "Linked Timer" title, which will allow to change it in place.

**Reply by nitsanavni** [reply]
If you like we can find the time to pair/ensemble on it together.
